### DISCUSSION TITLE
Are there any examples of rainbow highlighting using tree-sitter grammars?

### DISCUSSION THREAD

**Discussion by ahelwer** [tree-sitter/tree-sitter#1240] [Q&A] [answered]
By rainbow highlighting I mean coloring nestable matching delimiters - parentheses, brackets, even `\begin{e}` and `\end{e}` tokens in LaTeX - different colors to highlight their counterpart. For example, [this](https://github.com/luochen1990/rainbow). Anyone know of a project that does it?

**Comment by sogaiu** [comment, marked as answer]
Specific to Clojure and uses an older version of tree-sitter-clojure, but FWIW: https://github.com/sogaiu/vscode-clojure-colorizer
